vite.config.js

```diff
@@ -1,14 +1,23 @@
 // vite.config.js
 import { resolve } from "path";
+import browserslistToEsbuild from "browserslist-to-esbuild";
 import { defineConfig } from "vite";
+import topLevelAwait from "vite-plugin-top-level-await";
 
-export default defineConfig({
-  build: {
-    rollupOptions: {
-      input: {
-        main: resolve(__dirname, "index.html"),
-        maps: resolve(__dirname, "maps.html"),
+export default defineConfig(() => {
+  return {
+    root: resolve(__dirname, "website"),
+    build: {
+      target: browserslistToEsbuild(),
+      outDir: resolve(__dirname, "dist"),
+      emptyOutDir: true,
+      rollupOptions: {
+        input: {
+          main: resolve(__dirname, "website/index.html"),
+          maps: resolve(__dirname, "website/maps.html"),
+        },
```

Comment:
```suggestion
          maps: resolve(__dirname, "website/maps.html"),
          umd: resolve(__dirname, "website/umd.html"),
          es: resolve(__dirname, "website/es.html"),
        },
```

Always add new nested pages in the vite config to build the website